Mathlib/Analysis/PSeries.lean

```diff
@@ -223,8 +222,7 @@ theorem summable_nat_pow_inv {p : ℕ} :
 if and only if `1 < p`. -/
 theorem summable_one_div_nat_pow {p : ℕ} :
     Summable (fun n => 1 / (n : ℝ) ^ p : ℕ → ℝ) ↔ 1 < p := by
-  -- porting note (#10745): explicitly supplied two simp lemmas
-  simp only [one_div, Real.summable_nat_pow_inv]
+  simp
```

Comment:
A `simp only` with specified lemmas is more robust than just `simp`. Just remove the porting note.

Reply:
Sure, I will change it if you think that is better. Of course it is also more verbose...

Reply:
Done